backend/README.md

```diff
@@ -0,0 +1,43 @@
+## RUNNING THE BACKEND SERVER
+
+This README provides essential information for setting up and running the server.ts file in this project.
+
+### Prerequisites
+
+Before running the server.ts file, make sure you have the following prerequisites installed:
+
+-   **Node.js:** Ensure that you have Node.js installed on your machine. You can download it from [Node](https://nodejs.org/).
+
+-   **TypeScript:** Install TypeScript globally using the following command:
+
+    ```bash
+    npm install -g typescript
+    ```
+
+-   **TypeScript dependencies:** You can install them using the following command:
+
+    ```bash
+    npm install --save-dev typescript @types/node
```

Comment:
is not this already installed by the npm install? I am not so sure, could u please check it removing node_modules and running the command?

Reply:
You are right - running the npm i creates a subfolder node in the @types inside the node_modules. I have removed the redundant instruction from the readme.

Reply:
You are right - redundant instruction is gone from the readme file.